Disable consul update check

Disable the consul update check.

```
2020-07-21T19:08:54.397Z [ERROR] agent: Failed to check for updates: error="Get https://checkpoint-api.hashicorp.com/v1/check/consul?arch=amd64&os=linux&signature=a930e0f0-772d-f78f-9c76-1c297d56d6ab&version=1.7.2: dial tcp 23.23.113.197:443: connect: connection refused"
```

According to this (https://redmine.ixsystems.com/issues/27383) there is a `--disable_update_check` switch.

--- a/config/00-consul.json
+++ b/config/00-consul.json
@@ -1,3 +1,4 @@
 {
-    "enable_local_script_checks": true
+    "enable_local_script_checks": true,
+    "disable_update_check": true
 }

